DependencyInjection/Compiler/FormFactoryCompilerPass.php

```diff
@@ -11,10 +11,10 @@
 
 namespace Sonata\CoreBundle\DependencyInjection\Compiler;
 
-use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\FormPass;
```

Comment:
> There is no bundle with no requirement on fwb

I don't get what you're saying, but if we reference fwb somewhere in our code, then we should express what version constraints we have, right?